src/mixer.cpp

```diff
@@ -160,6 +153,10 @@ void Mixer::mix_output()
     commands.y *= RF_.params_.get_param_int(PARAM_ELEVATOR_REVERSE) ? -1 : 1;
     commands.z *= RF_.params_.get_param_int(PARAM_RUDDER_REVERSE) ? -1 : 1;
   }
+  else if(commands.F < RF_.params_.get_param_float(PARAM_MOTOR_IDLE_THROTTLE))
```

Comment:
We only want this behavior for multirotors, right? (Fixed-wings should be able to control rudder at zero throttle)